woeip/templates/__base.html

```diff
@@ -0,0 +1,7 @@
+{% include 'includes/_head.html' %}
```

Comment:
The file name only needs one underscore: `_base.html`.

Reply:
I've seen and used the style of one underscore in the past. For this project, I looked for established naming guidelines and couldn't find them. I stumbled on a [blog from 2012](https://oncampus.oberlin.edu/webteam/2012/09/architecture-django-templates) that recommended naming *includable* templates with a single underscore and *extendable* templates with a double underscore. I'd like to try this naming convention, unless they're outdated and other guidelines have become standardized.